source/SnapDump-Client/SDHTTPStore.class.st

```diff
@@ -116,11 +116,20 @@ SDHTTPStore >> shouldReportSnapshot: snapshot [
 { #category : #accessing }
 SDHTTPStore >> snapshotDumpFor: aSnapshot [ 
 	^ self client
-		headerAt: 'Accept' put: 'application/X-SnapDump-Fuel';
+		headerAt: 'Accept' put: SDStore mimeTypeFuel;
 		url: (self urlForSnapshot: aSnapshot);
 		get
 ]
 
+{ #category : #accessing }
+SDHTTPStore >> snapshotStackFor: aSnapshot [ 
+
+	^ (self client
+		headerAt: 'Accept' put: SDStore mimeTypeStack;
+		url: (self urlForSnapshot: aSnapshot);
+		get) utf8Decoded
+]
+
```

Comment:
At first I found confusing that there are two methods on the client but only one endpoint on the server checking on the mimeType. It's ok but I find it confusing that the client has to use "utf8Decoded" on the response. Why isn't thqt done on server side ?

Reply:
The server uses a streaming entity which means it does not read to whole file in memory before responding but streams it. And it sets a binary mime-type. It is possible to make it different but as it is done with the client classes I don't see a big problem here